packages/fxa-shared/metrics/amplitude.ts

```diff
@@ -341,93 +347,98 @@ export const amplitude = {
      *                            ease by perusing the code.
      */
     return (event: { [key: string]: any }, data: EventData) => {
-      if (!event || !data) {
-        return;
-      }
-
-      let eventType = event.type;
-      let mapping = events[eventType];
-      let eventCategory, eventTarget;
-
-      if (!mapping) {
-        for (const [key, value] of fuzzyEvents.entries()) {
-          const match = key.exec(eventType);
-          if (match) {
-            mapping = value;
+      try {
```

Comment:
This looks like a lot, but it's just the try catch, everything else is white space.